### DISCUSSION TITLE
In-game V-sync and Framerate limiter setting for lowest latency

### DISCUSSION THREAD

**Discussion by 1PercentSync** [LizardByte/community#231] [Sunshine] [answered]
should I turn on/off vsync ingame, limit not no limit for framerate to get lowest latency in sunshine?

**Comment by KuleRucket** [comment, marked as answer]
It depends.

Normal advice is to switch it off and only enable vsync in moonlight to avoid frame pacing issues. However this has nothing to do with latency. 

But on my AMD 7900 XTX, I find that capture/encode performance gets bad if the GPU is maxed out so I keep vsync or a frame rate cap on in-game to reduce the GPU usage and less latency. I didn't have this issue with nvidia since the encoder seems to run much more independently that the usual GPU functions.
